### DISCUSSION TITLE
v3.3.3 - v3.3.4

### DISCUSSION THREAD

**Discussion by peterlau** [Netflix/conductor#2545] [Announcements]
# What’s Changed

* fix issue that workflow not refetched when search with same query conditions (#2520)
* Fix Task Queue linking losing stack (stack not used on OSS).
* Add restart with latest definitions to bulk Action
* Fix useLatestDefinition_s_ param typo
* Add Domain to Task Details
* URL persisted lookback in search
* Add blank customTypeRenderers
* Fix auto refresh after workflow action (terminate/restart etc)

<hr /><em>This discussion was created from the release <a href='https://github.com/Netflix/conductor/releases/tag/v3.3.3'>v3.3.3</a>.</em>

**Comment by peterlau** [comment]
Please use 3.3.4 as 3.3.3 has a UI build issue.

**Reply by taoj-action** [reply]
Please add PR links to those items if possible. For example, I am especially interested to know where "Add Domain to Task Details" is implemented. Thanks!

**Reply by peterlau** [reply]
The specific commit for this feature was collapsed in a squash merge but the changes are as follows:
https://github.com/Netflix/conductor/blob/main/ui/src/pages/execution/Execution.jsx#L127
https://github.com/Netflix/conductor/blob/main/ui/src/pages/execution/TaskSummary.jsx#L17

**Reply by Lijiancheng668** [reply]
Hello, I want to know the conductor server that defines workflow after version 3. Does this server support jdk8? I also want to use springboot. Thank you
